visitors/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from users.models import Resident
+
+
+# Create your models here.
+class Visitor(models.Model):
+    name = models.CharField(max_length=100)
+    lastname = models.CharField(max_length=100)
+    rut = models.CharField(max_length=10)  # 01.234.567-9
```

Comment:
I think the format is without separating dots
```suggestion
    rut = models.CharField(max_length=10)  # 01234567-9
```